tests/test_infer/test_ops/triton/test_context_attn_unpad.py

```diff
@@ -19,8 +20,31 @@
 HEAD_DIM = 32
 
 
+def _fill_with_neg_inf(t):
+    return t.float().fill_(float("-inf")).type_as(t)
+
+
+# alibi mask calculation adapted from https://huggingface.co/baichuan-inc/Baichuan2-13B-Chat/blob/main/modeling_baichuan.py
```

Comment:
Same as above, use ref transformers v4.36 if it has this function

Reply:
This function is adapted from Baichuan modeling.